cmsl1t/plotting/efficiency.py

```diff
@@ -227,9 +232,15 @@ def make_fit(labels):
 
     def __make_overlay(self, pileup, threshold, hists, fits, labels, header):
         with preserve_current_style():
+            name = self.filename_format.format(pileup=pileup,
+                                               threshold=threshold)
             # Draw each efficiency (with fit)
-            canvas = draw(hists, draw_args={"xtitle": self.offline_title,
-                                            "ytitle": "Efficiency"})
+            draw_args = {"xtitle": self.offline_title, "ytitle": "Efficiency"}
+            # TODO: special case should not be implemented here!
```

Comment:
You mean like `# TODO: Remove when we update rootpy to >0.9.2` :)

I will put it in as it will force me to create the issue. Miss the gitlab feature "resolve with issue"

Reply:
Sorry, I've no objection to `TODO` comments in the code, I meant let's take out the actual code below this comment that the comment was referring to.  That's the sort of issue I want to avoid creeping in, not an issue of having TODO comments in there!

Reply:
Ah, that makes more sense ;).

However, at the moment we do not have a better solution, do we?

Reply:
I was thinking we could take it out of the PR, then let Aaron or Alex put it back in on their branch alone, or uncommitted until we come up with a better solution.  If we do let it in, we'd want a _very_ specific issue to be opened, I suppose

Reply:
The issue I had in mind here is to inject (like analyzers & filters in the new model) Styles into plotting routines.
This is something that needs to be done on our side and I am not sure about the exact form yet.
But having this code in master will force us to do something about it.

Reply:
Actually, issue already exists :) 
https://github.com/cms-l1t-offline/cms-l1t-analysis/issues/109